src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task{
+
+    protected String by;
+    public Event(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+
```

Comment:
Again, spare lines like  these may not be necessary as you have a space between the methods already.